transformer/processors.go

```diff
@@ -84,5 +86,21 @@ func processPathMappings(pms []transformertypes.PathMapping, sourcePath, outputP
 			}
 		}
 	}
+
+	for _, pm := range pms {
+		destPath := pm.DestPath
+		if !filepath.IsAbs(pm.DestPath) {
+			destPath = filepath.Join(outputPath, pm.DestPath)
+		}
+		switch strings.ToLower(pm.Type) {
+		case strings.ToLower(transformertypes.DeletePathMappingType):
```

Comment:
Do we need a switch case for this? Why not just a if with Equalsfold?

Reply:
done.